### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by morfo2** [G1-R0/IPTV-stremio-addon#1]
The website is down. Did you quit the project?

**Comment by gllsrssx** [comment]
> The website is down. Did you quit the project?

It works but I took it online since I wasn't using it anymore. I can put it back online if you wish. You can also run it locally

**Comment by gllsrssx** [comment]
> The website is down. Did you quit the project?

I have put it back online. note that it might take a while for the first startup to awake the server.

on a later date I might also update this project so it looks better

**Comment by GSD2112** [comment]
When I paste the addon link it doesn't work :/

**Comment by KiLoS102** [comment]
I have other problem... When im trying open "Configure", there im getting new website "Cannot GET /configure" :(

**Comment by gllsrssx** [comment]
yes, I dont maintain it anymore since there are good free options and there is no demand.
if you are looking for a certain country/language, please let me know.
